### PR TITLE
Moving angular material to master to be accessible at syncids.github.io

### PR DESCRIPTION
**Pull request by ntgn81**
Old site is at https://syncids.github.io/old

### CODE REVIEW

**Comment by ntgn81** [comment]
Just moving things around

File: app/components/documents/documents.directive.js [hunk 1]
```diff
@@ -19,11 +19,39 @@
...
+    function initialize() {
+      $scope.$watch('vm.docs', function(newDocs) {
+        if (!newDocs) return;
+
+        newDocs.us = newDocs.us || [];
+        newDocs.foreign = newDocs.foreign || [];
+        newDocs.other = newDocs.other || [];
+
+        _.each(newDocs, function(docs) {
+          if (!vm.matter) return;
```
Comment (erlikh) [inline app/components/documents/documents.directive.js]: Seems like we can check it outside of the iterator

File: app/components/documents/documents.directive.js [hunk 2]
```diff
@@ -33,5 +61,65 @@
...
+        // update
+        return;
+      }
+
+      // add
+      $timeout(function() {
+        doc.id = new Date().getTime();
+        vm.docs.foreign.push({});
+      }, 300);
+    }
```
Comment (erlikh) [inline app/components/documents/documents.directive.js]: Seems pretty same with the usDocChanged

File: app/components/documents/documents.directive.js [hunk 3]
```diff
@@ -33,5 +61,65 @@
...
+        // update
+        return;
+      }
+
+      // add
+      $timeout(function() {
+        doc.id = new Date().getTime();
+        vm.docs.other.push({});
+      }, 300);
+    }
```
Comment (erlikh) [inline app/components/documents/documents.directive.js]: That one too.

**Pull request merged**